### PR TITLE
Fix the following typos:

### PR DESCRIPTION
**Pull request by elianerpereira**
import - imported
semver - server
provides - provider
provide - provide
Remove `describes`
this - these
initalization - initialization
Simpliflied - Simplified

### CODE REVIEW

File: docs/user-guide/01-blueprint-reference.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ version = "0.0.1"
...
 - The `description` attribute is a string that can be a longer description of the blueprint and is only used for display purposes.
-- The `version` attribute is a string that contains a semver compatible version number. If a new blueprint is uploaded with the same version the server will automatically bump the PATCH level of the version. If the version doesn't match it will be used as is. For example, uploading a blueprint with version set to 0.1.0 when the existing blueprint version is 0.0.1 will result in the new blueprint being stored as version 0.1.0.
+- The `version` attribute is a string that contains a server compatible version number. If a new blueprint is uploaded with the same version the server will automatically bump the PATCH level of the version. If the version doesn't match it will be used as is. For example, uploading a blueprint with version set to 0.1.0 when the existing blueprint version is 0.0.1 will result in the new blueprint being stored as version 0.1.0.
```
Comment (ochosi) [inline docs/user-guide/01-blueprint-reference.md]: It really should be "semver" as in "semantic versioning", not server. However, let's not use the abbreviation but instead "semantic versioning" for more clarity.

File: docs/user-guide/01-blueprint-reference.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ The `packages` and `modules` lists contain objects with a `name` and optional `v
 
 *Currently there are no differences between packages and modules in `osbuild-composer`. Both are treated like an rpm package dependency.*
 
-> When using virtual provides as the package name the version glob should be `*`. And be aware that you will be unable to `freeze` the blueprint. This is because the provide will expand into multiple packages with their own names and versions.
+> When using virtual providers as the package name the version glob should be `*`. And be aware that you will be unable to `freeze` the blueprint. This is because the provider will expand into multiple packages with their own names and versions.
```
Comment (ochosi) [inline docs/user-guide/01-blueprint-reference.md]: I think this should actually be "provides". In virtual packages, this keyword is used to list all the packages provided by a virtual package.
Maybe we could make it clearer by using markdown for monospace formatting `provides` to make it visually stand out as a technical term?

**Review by ochosi — CHANGES_REQUESTED** [review]
This is awesome - thanks for catching so many typos!
I think there are two false positives there, but either also warrants improvements for more clarity.

File: docs/user-guide/01-blueprint-reference.md (hunk 1 above)
Comment (elianerpereira) [reply docs/user-guide/01-blueprint-reference.md]: Thank you for clarifying this. I will fix it.

File: docs/user-guide/01-blueprint-reference.md (hunk 2 above)
Comment (elianerpereira) [reply docs/user-guide/01-blueprint-reference.md]: Thank you for clarifying this. I will fix it and add the correct markup.

File: docs/user-guide/01-blueprint-reference.md [hunk 3]
```diff
@@ -13,9 +13,9 @@ version = "0.0.1"
...
 - The `description` attribute is a string that can be a longer description of the blueprint and is only used for display purposes.
-- The `version` attribute is a string that contains a semver compatible version number. If a new blueprint is uploaded with the same version the server will automatically bump the PATCH level of the version. If the version doesn't match it will be used as is. For example, uploading a blueprint with version set to 0.1.0 when the existing blueprint version is 0.0.1 will result in the new blueprint being stored as version 0.1.0.
+- The `version` attribute is a string that contains a server compatible version number. If a new blueprint is uploaded with the same version the semantic versioning will automatically bump the PATCH level of the version. If the version doesn't match it will be used as is. For example, uploading a blueprint with version set to 0.1.0 when the existing blueprint version is 0.0.1 will result in the new blueprint being stored as version 0.1.0.
```
Comment (ochosi) [inline docs/user-guide/01-blueprint-reference.md]: Sorry, but this is now doubly wrong 😅

**Review by ochosi — CHANGES_REQUESTED** [review]
Thanks for fixing the one, one left to go

File: docs/user-guide/01-blueprint-reference.md (hunk 3 above)
Comment (elianerpereira) [reply docs/user-guide/01-blueprint-reference.md]: Sorry, fixed, hopefully :D

**Review by ochosi — APPROVED** [review]
Great, thanks for all the fixes!

**Pull request merged**